contracts/contracts/coordination/Coordinator.sol

```diff
@@ -525,6 +556,125 @@ contract Coordinator is Initializable, AccessControlDefaultAdminRulesUpgradeable
         processReimbursement(initialGasLeft);
     }
 
+    function handoverRequest(
+        uint32 ritualId,
+        address departingParticipant,
+        address incomingParticipant
+    ) external onlyRole(HANDOVER_SUPERVISOR_ROLE) {
+        require(isRitualActive(ritualId), "Ritual is not active");
+
```

Comment:
We need to validate that `departingParticipant` is in this ritual, and that `incomingParticipant` is not.

Reply:
✔️